noir-projects/noir-protocol-circuits/crates/types/src/constants.nr

```diff
@@ -33,12 +33,8 @@ global MAX_NOTE_HASH_READ_REQUESTS_PER_CALL: u64 = 32;
 global MAX_NULLIFIER_READ_REQUESTS_PER_CALL: u64 = 2; // Change it to a larger value when there's a seperate reset circuit.
 global MAX_NULLIFIER_NON_EXISTENT_READ_REQUESTS_PER_CALL: u64 = 2;
 global MAX_NULLIFIER_KEY_VALIDATION_REQUESTS_PER_CALL: u64 = 1;
-// Note: if changing enc logs per call value by n, change PRIVATE_CALL_STACK_ITEM_LENGTH and PRIVATE_CIRCUIT_PUBLIC_INPUTS_LENGTH by 2n
-// and change DEPLOYER_CONTRACT_ADDRESS to new value
-global MAX_ENCRYPTED_LOGS_PER_CALL: u64 = 4;
-// Note: if changing unenc logs per call value by n, change PRIVATE_CALL_STACK_ITEM_LENGTH, PRIVATE_CIRCUIT_PUBLIC_INPUTS_LENGTH, and PUBLIC_CIRCUIT_PUBLIC_INPUTS_LENGTH by 2n
-// and change DEPLOYER_CONTRACT_ADDRESS to new value
-global MAX_UNENCRYPTED_LOGS_PER_CALL: u64 = 4;
+global MAX_ENCRYPTED_LOGS_PER_CALL: u64 = 4; // If modifying, update DEPLOYER_CONTRACT_ADDRESS.
```

Comment:
I feel the "update deployer address" applies to way too many constants in here :-(